packages/ckeditor5-build-helpjuice-v2/src/plugins/glossary/glossaryui.js

```diff
@@ -1,67 +1,67 @@
-import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
-import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import { ButtonView } from 'ckeditor5/src/ui';
+import { Plugin } from 'ckeditor5/src/core';
 import BookIcon from './icons/book-icon.svg';
 
 export default class GlossaryUI extends Plugin {
 	init() {
 		const editor = this.editor;
 		const t = editor.t;
 
-		editor.ui.componentFactory.add('glossary', locale => {
-			const buttonView = new ButtonView(locale);
+		editor.ui.componentFactory.add( 'glossary', locale => {
+			const buttonView = new ButtonView( locale );
 
-			buttonView.set({
-				label: t('Insert Glossary'),
+			buttonView.set( {
+				label: t( 'Insert Glossary' ),
 				icon: BookIcon,
 				tooltip: true,
-				class: "insert-glossary-btn"
-			});
+				class: 'insert-glossary-btn'
+			} );
 
 			// Execute the command when the button is clicked (executed).
-			this.listenTo(buttonView, 'execute', () => {
-				const glossaryModal = document.getElementById("new-glossary-term-modal");
-				let glossaryTermExpression = glossaryModal.querySelector("#glossary_term_expression");
-				let glossaryTermDefinition = glossaryModal.querySelector("#glossary_term_definition");
+			this.listenTo( buttonView, 'execute', () => {
+				const glossaryModal = document.getElementById( 'new-glossary-term-modal' );
+				const glossaryTermExpression = glossaryModal.querySelector( '#glossary_term_expression' );
+				const glossaryTermDefinition = glossaryModal.querySelector( '#glossary_term_definition' );
 
-				const range = editor.model.document.selection.getFirstRange()
-				let selection = "";
-				for (const item of range.getItems()) {
-					if (item.data) {
+				const range = editor.model.document.selection.getFirstRange();
+				let selection = '';
+				for ( const item of range.getItems() ) {
```

Comment:
Can you fix your formatting? Why the extra spaces in several lines? Is this your editor?

Reply:
Hi @diego-aslz! It's the [.eslintrc.js](https://github.com/Helpjuice/ckeditor5/blob/helpjuice-build-v2/.eslintrc.js) and [.editorconfig](https://github.com/Helpjuice/ckeditor5/blob/helpjuice-build-v2/.editorconfig) files that are causing those formatting issues. Maybe we should update them to the same config we use in the helpjuice repo.

Reply:
This is eslint then, our editorconfig is not setup like this. Can we fix the linter?

Reply:
I think that's a good idea. Up to you if you wanna do that separately, though.